docs/upgrade-sdk-to-mux.md

```diff
@@ -0,0 +1,153 @@
+# How to Upgrade a provider that has partially migrated to the Plugin Framework
+
+Follow these steps if you have a Pulumi provider that was bridged from a Terraform
+provider built against [Terraform Plugin
+SDK](https://github.com/hashicorp/terraform-plugin-sdk) and you want to upgrade it to a
+version that has migrated some but not all resources/datasources to the [Plugin
+Framework](https://github.com/hashicorp/terraform-plugin-sdk?tab=readme-ov-file).
+
+1. Ensure you have access to the
+   `github.com/hashicorp/terraform-plugin-framework/provider.Provider` from the upstream
+   provider.  If the provider is shimmed (or needs to be), you can follow step (1) from
+   ["How to Upgrade a Bridged Provider to Plugin Framework"](../pf/README.md#How to
```

Comment:
is the link here broken? the review does not render it proerly

Reply:
Fixed.